torch/distributed/tensor/parallel/api.py

```diff
@@ -82,6 +87,14 @@ def parallelize_module(  # type: ignore[return]
 
     torch._C._log_api_usage_once("torch.distributed.tensor.parallel.parallelize_module")
 
+    # instantiate a TP RNG state tracker if it's not there
+    if (
+        is_rng_supported_mesh(device_mesh) and
+        not isinstance(random._rng_tracker, TensorParallelRNGTracker)
```

Comment:
Can we also pass in the Tracker in case users want to use OffsetBasedRNGTracker?

Reply:
then we need a way to control it (e.g. a switch to decide which tracker to use for TP). Maybe a config variable.

Reply:
I mean passing in as a input variable of the API.